test/test_with_xpress.jl

```diff
@@ -1025,6 +1027,29 @@ end
     # Test generator outputs
     @test typeof(r) == Model # this is true when the model is infeasible
 
+    ### Scenario 3: Indonesia. Wind (custom prod) and Generator only
+    m = Model(optimizer_with_attributes(Xpress.Optimizer, "MIPRELSTOP" => 0.01, "OUTPUTLOG" => 0))
+    post_name = "wind_intl_offgrid.json" 
+    post = JSON.parsefile("./scenarios/$post_name")
+    post["ElectricLoad"]["loads_kw"] = [10.0 for i in range(1,8760)]
+    scen = Scenario(post)
+    df = CSV.read("./data/example_wind_prod_factor_kw.csv", skipto=2, DataFrame)
```

Comment:
please remove DataFrames and CSV from the test environment and use [readdlm](https://docs.julialang.org/en/v1/stdlib/DelimitedFiles/#DelimitedFiles.readdlm-Tuple{Any,%20AbstractChar}) from the standard library DelimitedFiles instead

Reply:
removed in https://github.com/NREL/REopt.jl/pull/75/commits/840ea0ce8d3fa56c2ff5fa01b13d06ba8a85854d